CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+## 0.0.1-dev
```

Comment:
Dart follows [sematic versioning](https://semver.org/), where pre-release versions are incremented after the `-dev` tag.
For example: 

1. Initially: `0.0.1-dev.1`
2. Next pre-release will be `0.0.1-dev.2`
3. When all of these are merged to master (stable version), it should be `0.0.1`

Reply:
My bad going to have to leave this as i've already published to pub.dev